src/cli.js

```diff
@@ -1,68 +1,83 @@
-      boolean: true,
-      requiresArg: false,
-      required: false
-    },
-    report: {
-      alias: 'r',
-      description: 'Output report (Default: false)',
-      requiresArg: false,
-      boolean: true,
-      required: false
-    },
-    list: {
-      alias: 'l',
-      description: 'List all benchmarks',
-      requiresArg: false,
-      required: false,
-      boolean: true
-    },
-    grep: {
-      alias: 'g',
-      description: '<regexp> Regular expression used to match benchmarks (Default: /.*/)',
-      requiresArg: true,
-      required: false
-    },
-    stressLimit: {
-      description: '<Int or Infinity> seconds to run a stress benchmark (Default: 300)',
-      requiresArg: true,
-      required: false
-    },
-    baselineLimit: {
-      description: '<Int> benchmark cycle limit for baseline benchmarks (Default: 1000)',
-      requiresArg: true,
-      required: false
-    },
-    logLimit: {
-      description: '<Int> max log size used for baseline benchmarks (inclusive) (Default: 10000)',
-      requiresArg: true,
-      required: false
-    }
+let { benchmarks, output, node, browser, baselines } = program.opts()
+
+benchmarks = path.resolve(benchmarks)
+if (!existsSync(benchmarks)) throw new Error(`benchmarks path does not exist: ${benchmarks}`)
+
+if (output) output = path.resolve(output)
+if (baselines) baselines = path.resolve(baselines)
+
+const { mkdtempSync } = require('fs')
+const os = require('os')
+const tmpdir = mkdtempSync(path.join(os.tmpdir(), 'orbit-db-benchmark_'))
+
+// get benchmark file paths
+const { execSync } = require('child_process')
+const benchmarkPaths = statSync(benchmarks).isDirectory()
+  ? execSync('ls -1 *.benchmark.js', { cwd: benchmarks })
+    .toString()
+    .split('\n')
+    .filter(a => a)
+    .map(p => path.join(benchmarks, p))
+  : [benchmarks]
+
+// benchmarker server, collects logs and results
+const benchmarkerServer = require('./benchmarker/server.js').create()
+
+// webpack port; reuse same port for all benchmarks to keep IndexedDB same
+let webpackPort = null
+const getPort = () => new Promise((resolve, reject) => {
+  const server = net.createServer()
+  server.unref()
+  server.on('error', reject)
+  server.listen(0, () => {
+    const { port } = server.address()
+    server.close(() => resolve(port))
   })
-  .example('$0 -r -g append-baseline', 'Run a single benchmark (append-baseline)')
-  .example('$0 -r -g values-.*-baseline', 'Run all of the values baseline benchmarks')
-  .argv
+})
+
+// const getBenchmarkHook = require('./get-benchmark-hook')
+const { Worker } = require('worker_threads')
 
-// Was this called directly from the CLI?
-// For mocha tests - maybe we can mitigate this but for now it works
-if (require.main === module) {
-  try {
-    const benchmarks = require(process.cwd() + '/benchmarks')
-    start(benchmarks, argv)
-  } catch (e) {
-    throw new Error(e.message)
+async function execBenchmarks (browser) {
```

Comment:
for clarity let's either rename this variable to `isBrowser` for convention or make it a named property i.e. `execBenchmarks({ browser })`